### PR TITLE
Update Main_EDA.Rmd

### PR DESCRIPTION
**Pull request by Kydoimos97**

### CODE REVIEW

File: EDA/Main_EDA.Rmd [hunk 1]
```diff
@@ -217,12 +218,23 @@ Table 1 shows that after mapping the Item_desc variables to the main dataset the
...
 
+## Temprature NA's
+
+```{r}
+new_df <- df[is.na(df$maxtemp),]
+
+levels(as.factor(new_df$site_id))
+
+levels(as.factor(new_df$date))
+```
```
Comment (kelsey-codes) [inline EDA/Main_EDA.Rmd:230]: This deeper dive into the NAs show that the missing temperature data is randomly scattered across dates and sites, and we cannot point to a single factor for why the information is missing.  We will need to perform further analysis during data cleaning to determine whether we will impute or remove the NAs.

Comment (kelsey-codes) [inline EDA/Main_EDA.Rmd:230]: ``
This deeper dive into the NAs show that the missing temperature data is randomly scattered across dates and sites, and we cannot point to a single factor for why the information is missing. We will need to perform further analysis during data cleaning to determine whether we will impute or remove the NAs.
''

**Review by kelsey-codes — APPROVED** [review]

File: EDA/Main_EDA.Rmd [hunk 2]
```diff
@@ -213,21 +214,29 @@ tableContinuous(df[,sapply(df, is.numeric)],
...
+This deeper dive into the NAs show that the missing temperature data is randomly scattered across dates and sites, and we cannot point to a single factor for why the information is missing. We will need to perform further analysis during data cleaning to determine whether we will impute or remove the NAs.
+\newpage
 
-**The table output I see has sq_footage, locale, and mpds no quantity sold or sales**
+#### Factorized
 
-Table 2 shows that Quantity sold and sales seem to have extremely long right tails. Since Sales is the target variable this needs to be further investigated and looked at to ensure optimal algorithm performance. The Project variable holds limited information since only 0.02% of the data entries deviate from the baseline of NONE.
+```{r, results="asis"}
+# summarize factorized variables
+tableNominal(df[c(12,13,14,15)], cap = "Summary of Factorized variables", cumsum = FALSE)
```
Comment (Kydoimos97) [inline EDA/Main_EDA.Rmd:237]:
```suggestion
tableNominal(df[c(12,13,14,15)], cap = "Summary of Factorized variables", cumsum = FALSE, comment = FALSE)

```

File: EDA/Main_EDA.Rmd [hunk 3]
```diff
@@ -204,6 +201,10 @@ names(df) <- tolower(make.names(names(df)))
 rm(products, candy_vector, id_vector)
 ```
 
+### Summary Tables
+
+#### Numeric
+
 ```{r, results="asis"}
 # summarize numeric variables
 tableContinuous(df[,sapply(df, is.numeric)],
```
Comment (Kydoimos97) [inline EDA/Main_EDA.Rmd:210]:
```suggestion
tableContinuous(df[,sapply(df, is.numeric)], , comment = FALSE,

```

**Review by kelsey-codes — APPROVED** [review]

**Review by Kydoimos97 — COMMENTED** [review]
Formatting

**Pull request merged**